CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # kitchen-azurerm Changelog
 
+## [0.14.10] - 2019-10-21
```

Comment:
Please remove this file from the PR, I'll deal with the Changelog at release time. Thanks!

Reply:
Definitely.